package.json

```diff
@@ -6,15 +6,16 @@
   "main": "index.js",
   "author": "Harris Borawski",
   "scripts": {
+    "build": "lerna run build",
     "test": "jest --coverage",
     "test:watch": "jest --watch",
     "version": "git add .",
-    "lint": "eslint '**/*.{js,jsx}'",
+    "lint": "eslint '**/*.{js,jsx}' --ignore-pattern '**/lib/**/*'",
```

Comment:
This _should_ be covered by the [.eslintignore](https://github.com/intuit/ReplayWeb/blob/master/.eslintignore) file.